pkgs/by-name/pi/pid-fan-controller/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  rustPlatform,
+  fetchFromGitHub,
+  lib,
+}:
+let
+  pver = "0.1.1";
+in
+rustPlatform.buildRustPackage {
+  pname = "pid-fan-controller";
+  version = pver;
```

Comment:
```suggestion
let
  pname = "pid-fan-controller";
  version = "0.1.1";
in
rustPlatform.buildRustPackage {
  inherit pname version;
```

Reply:
whats the rational for factoring `pname` out? It will not change after all.

Reply:
No real reason. Some people prefer to just have `version` inside the `let...in` statement, and some people prefer to have everything, including `src` and `meta` in the `let...in`. Feel free to do whatever you think is best.

Reply:
Hey, you marked this as resolved, but `pver` has not been removed. `pver` is not a standard convention; please change it to `version` and `inherit version;`